### PR TITLE
Spike login flow change

### PR DESCRIPTION
**Pull request by jsugarman**
change login flow to interject a confirmation page for profiles that could be duplicates.

### CODE REVIEW

File: spec/controllers/concerns/shared_examples_for_session_person_creator.rb [hunk 1]
```diff
@@ -66,9 +73,9 @@ def flashable_expectation_stub
     end
 
     context 'for a new person' do
-      before do
-        flashable_expectation_stub
-      end
+      # before do
+      #   flashable_expectation_stub
+      # end
```
Comment (cleargif) [inline spec/controllers/concerns/shared_examples_for_session_person_creator.rb]: @jsugarman are you committing these comments on purpose?

File: spec/controllers/concerns/shared_examples_for_session_person_creator.rb [hunk 2]
```diff
@@ -80,13 +87,14 @@ def flashable_expectation_stub
...
+
       it_behaves_like 'new person created' do
```
Comment (cleargif) [inline spec/controllers/concerns/shared_examples_for_session_person_creator.rb]: @jsugarman

**Comment by cleargif** [comment]
The FE and CSS are good to go -  I did inline comment on a couple of things though

File: spec/controllers/concerns/shared_examples_for_session_person_creator.rb (hunk 2 above)
Comment (jsugarman) [inline spec/controllers/concerns/shared_examples_for_session_person_creator.rb]: cheers, no, was reminder to spec some flashes. will remove

**Pull request merged**